app/assets/stylesheets/layout/common/_header.scss

```diff
@@ -1,7 +1,7 @@
 .l-header {
-  background: $color-green-primary;
-  min-height: $baseline-unit*11;
-  position: relative;
+  width: 100%;
+  z-index: 200;
+  min-height: 55px;
```

Comment:
Nonetheless I'll define the header height as $baseline-unit*9 (which is the equivalent according to my calculations) for the smaller viewport in order to maintain continuity.